### PR TITLE
use plotbot for the chart studio hosted plots

### LINKED ISSUE #1367: 404 Plot Twist in Python Getting Started
https://plot.ly/python/getting-started/

### PR DESCRIPTION
**Pull request by michaelbabyn**
resolves #1367

### CODE REVIEW

File: _posts/python/getting-started/2015-06-30-getting-started.html [hunk 1]
```diff
@@ -319,11 +333,11 @@ <h3 id="Start-Plotting-Online">Start Plotting Online<a class="anchor-link" href=
 
 <div class="output_area">
 
-    <div class="prompt"></div>
+<div class="prompt"></div>
 
 
 <div class="output_subarea output_stream output_stdout output_text">
-<pre>Help on function plot in module plotly.plotly.plotly:
+<pre>Help on function plot in module chart_studio.plotly.plotly:
```
Comment (nicolaskruchten) [inline _posts/python/getting-started/2015-06-30-getting-started.html]: Ah so this isn't going to work for the v3 docs you're editing now... you'll need to stick with `plotly.whatever` for PRs in this repo

Comment (michaelbabyn) [reply _posts/python/getting-started/2015-06-30-getting-started.html]: Whoops, I didn't notice that. I just re-ran the code using the PythonPlotBot account. I'll fix that now. I'm not sure why that was changed.

Comment (nicolaskruchten) [reply _posts/python/getting-started/2015-06-30-getting-started.html]: Maybe this is actually normal in Plotly.py 3.10.0 actually... Maybe worth checking in with @jonmmease about how things were orged/reorged before the v4_integration branch

**Comment by nicolaskruchten** [comment]
@michaelbabyn we can close this one out, right?

**Pull request closed without merging**